fortuna/data/loader.py

```diff
@@ -809,7 +809,7 @@ def __next__(self):
         self._ready = False
         return self._batch
 
-    def prefetch(self):
+    def _prefetch(self):
```

Comment:
Here's the fix to be in line with the expectation in line 808. I assume the method shall indeed be private to the generator.